specification/purview/Azure.Analytics.Purview.DataMap/routes.tsp

```diff
@@ -0,0 +1,2355 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-client-generator-core";
```

Comment:
We shouldn't import "@azure-tools/typespec-client-generator-core" outside the client.tsp file.

Reply:
Removed.